sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -0,0 +1,1988 @@
+    connectionString?: string;
+    query: string;
+    resourceId: string;
+    serviceName: string;
+}
+
+// @public
+export interface ApplicationInsightsConfigurationOutput extends InputDataOutputParent {
+    connectionString?: string;
+    query: string;
+    resourceId: string;
+    serviceName: string;
+    // (undocumented)
+    readonly type: "app_insights";
+}
+
+// @public
+export type AuthenticationTypeOutput = "ApiKey" | "AAD" | "SAS";
+
+// @public
+export interface AzureAISearchResource {
+    indexes?: Array<IndexResource>;
+}
+
+// @public
+export interface AzureAISearchResourceOutput {
+    indexes?: Array<IndexResourceOutput>;
+}
+
+// @public
+export interface AzureAISearchToolDefinition extends ToolDefinitionParent {
+    type: "azure_ai_search";
+}
+
+// @public
+export interface AzureAISearchToolDefinitionOutput extends ToolDefinitionOutputParent {
+    type: "azure_ai_search";
+}
+
+// @public
+export interface BingGroundingToolDefinition extends ToolDefinitionParent {
+    bing_grounding: ToolConnectionList;
+    type: "bing_grounding";
+}
+
+// @public
+export interface BingGroundingToolDefinitionOutput extends ToolDefinitionOutputParent {
+    bing_grounding: ToolConnectionListOutput;
+    type: "bing_grounding";
+}
+
+// @public
+export interface CodeInterpreterToolDefinition extends ToolDefinitionParent {
+    type: "code_interpreter";
+}
+
+// @public
+export interface CodeInterpreterToolDefinitionOutput extends ToolDefinitionOutputParent {
+    type: "code_interpreter";
+}
+
+// @public
+export interface CodeInterpreterToolResource {
+    data_sources?: Array<VectorStoreDataSource>;
+    file_ids?: string[];
+}
+
+// @public
+export interface CodeInterpreterToolResourceOutput {
+    data_sources?: Array<VectorStoreDataSourceOutput>;
+    file_ids?: string[];
+}
+
+// @public (undocumented)
+export interface ConnectionsOperations {
+    getConnection: (connectionName: string, requestParams?: OptionalRequestParameters) => Promise<GetConnectionResponseOutput>;
+    getConnectionWithSecrets: (connectionName: string, requestParams?: OptionalRequestParameters) => Promise<GetConnectionResponseOutput>;
+    getWorkspace: (requestParams?: OptionalRequestParameters) => Promise<GetWorkspaceResponseOutput>;
+    listConnections: (options?: ListConnectionsQueryParamProperties, requestParams?: OptionalRequestParameters) => Promise<Array<GetConnectionResponseOutput>>;
+}
+
+// @public
+export enum connectionToolType {
+    BingGrounding = "bing_grounding",
+    MicrosoftFabric = "microsoft_fabric",
+    SharepointGrounding = "sharepoint_grounding"
+}
+
+// @public
+export type ConnectionType = "AzureOpenAI" | "Serverless" | "AzureBlob" | "AIServices" | "CognitiveSearch";
+
+// @public
+export type ConnectionTypeOutput = "AzureOpenAI" | "Serverless" | "AzureBlob" | "AIServices" | "CognitiveSearch";
+
+// @public
+export interface CreateAgentOptions {
+    description?: string | null;
```

Comment:
Same here - was this autogen'd? having both `undefined` and `null` is not something I am used to seeing. Please reduce this to just `undefined` (`description?: string`), same for the rest

Reply:
Yes there are auto generated 

https://github.com/Azure/azure-sdk-for-js/blob/feature/azure-ai-sdk/sdk/ai/ai-projects/src/generated/src/models.ts